app/javascript/routes/routes.ts

```diff
@@ -0,0 +1,26 @@
+/* tslint:disable:object-literal-sort-keys */
```

Comment:
There's already this file in `app/javascript/views/routes.ts` should you use that one instead?

Reply:
I'm agreed we should have a single location for `routes`, could we keep it outside of views, in its own `routes` folder?